src/webgl/p5.RendererGL.js

```diff
@@ -1140,15 +1147,22 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
       // filterParameter uniform only used for POSTERIZE, and THRESHOLD
       // but shouldn't hurt to always set
       this.filterShader.setUniform('filterParameter', filterParameter);
-      pg.rect(-this.width/2, -this.height/2, this.width, this.height);
+      pg.rect(-this.width / 2, -this.height / 2, this.width, this.height);
 
     }
     // draw pg contents onto main renderer
     this._pInst.push();
+    pg._pInst.resetMatrix();
     this._pInst.noStroke();
-    this._pInst.image(pg, -this.width/2, -this.height/2,
+    pg._pInst.imageMode(constants.CORNER);
+    pg._pInst.blendMode(constants.BLEND);
+    this.clear();
+    this._pInst.push();
+    pg._pInst.setCamera(this.filterCamera);
```

Comment:
I think this one should be `this._pInst.setCamera(...)`, since we're using it for the `this._pInst.image` call below.

Reply:
https://github.com/processing/p5.js/assets/127239756/7abe0737-218f-4761-b1c8-cd39ac32aed0

this is the output.

Reply:
let me use this._pInst.setCamera()

Reply:
@davepagurek,  using this._pInst.setCamera() along with a reset matrix below it solves the issue to a significant extent. However, it causes changes in the image size and translations.
here is the editor:-

https://editor.p5js.org/aman12345/sketches/gfndC66Pf

Reply:
the canvas which was moved along, now it is solved.